compiler/rustc_session/src/config.rs

```diff
@@ -1441,6 +1441,7 @@ impl CheckCfg {
         // These three are never set by rustc, but we set them anyway: they
         // should not trigger a lint because `cargo doc`, `cargo test`, and
         // `cargo miri run` (respectively) can set them.
+        ins!(sym::clippy, no_values);
```

Comment:
The check-cfg section of the unstable book also needs updating: https://github.com/rust-lang/rust/blob/4ae1e79876fcee94a56c1edf7c029a677084d7fc/src/doc/unstable-book/src/compiler-flags/check-cfg.md?plain=1#L80-L87

Reply:
Good catch!

Reply:
The code here should have a comment mentioning the docs :)

Reply:
Adding it.